com.ibm.streamsx.iot/com.ibm.streamsx.iot.watson/devicesubscribe.spl

```
@@ -7,12 +7,14 @@ namespace com.ibm.streamsx.iot.watson;
 
 use com.ibm.streamsx.iot::*;
 use com.ibm.streamsx.topology.topic::Subscribe;
+use com.ibm.streamsx.json::JSONToTuple;
+use com.ibm.streamsx.json::Json;
 
 /**
  * Subsribes to device commands to be sent to devices for an organization.
  *
  * Subscribes to the topic `streamsx/iot/device/commands/send`
- * with a tuple type of [DeviceCmd].
+ * with a tuple type of [DeviceCmd], or a JSON string that can be converted to a [DeviceCmd]
```

Yes error handling should be added. JSONToTuple supports optional output port. In case of JSON errors a tuple is send out at this port ... and should be logged downstream. Alternative error handling of JSONToTuple is operator-fail or silently ignoring the error. Both are not appropriate here.